funktionale-try/src/test/kotlin/org/funktionale/tries/TryTest.kt

```diff
@@ -59,6 +59,11 @@ class TryTest {
         failure.foreach { fail() }
     }
 
+    @Test fun onEach() {
```

Comment:
Maybe there should be a test for `success.onEach { throw Exception("expected") }`